people/bhagyasri-canumalla.md

```diff
@@ -0,0 +1,7 @@
+Bhagyasri Canumalla
+-------------
+
+![](photos/bhagya.jpg)
```

Comment:
Please rename the JPG file to `bhagyasri-canumalla.jpg`, there might be other people with the same first name as you ;).